### PR TITLE
feat: Add react-markdown to display proposal info

### PR DESCRIPTION
**Pull request by Hemanthghs**

### CODE REVIEW

**Comment by aleem1314** [comment]
Lets try these solutions and see https://github.com/remarkjs/react-markdown/issues/635#issuecomment-956158474

**Comment by Hemanthghs** [comment]
> Lets try these solutions and see [remarkjs/react-markdown#635 (comment)](https://github.com/remarkjs/react-markdown/issues/635#issuecomment-956158474)

Okay Aleem

File: frontend/src/pages/gov/chain/ProposalInfo.js [hunk 1]
```diff
@@ -210,7 +218,8 @@ export default function ProposalInfo() {
           aminoConfig: chainInfo.aminoConfig,
           prefix: chainInfo.config.bech32Config.bech32PrefixAccAddr,
           feeAmount:
-            chainInfo.config?.feeCurrencies?.[0]?.gasPriceStep.average * 10 ** currency.coinDecimals,
+            chainInfo.config?.feeCurrencies?.[0]?.gasPriceStep.average *
```
Comment (charymalloju) [inline frontend/src/pages/gov/chain/ProposalInfo.js:221]: why shouldn't we write one utility function  **getAverageFee** to get the average fee?

**Comment by aleem1314** [comment]
Closing in favour of #675

**Pull request closed without merging**